components/encryption/src/manager/mod.rs

```diff
@@ -331,7 +421,7 @@ fn generate_data_key(method: EncryptionMethod) -> (u64, Vec<u8>) {
 #[derive(Clone)]
```

Comment:
let's remove the Clone trait, also remove Arc for `dicts`.

Reply:
removed